overrides/config/incontrol/loot.json

```diff
@@ -16,11 +16,7 @@
 
 "mob":"minecraft:iron_golem",
 
-"player":true,
-
-"item":"minecraft:iron_ingot",
-
-"itemcount": "0/0/0"
```

Comment:
no iron at all?? not sure I agree, we'll get people asking why don't golems drop iron but I do get not wanting people making iron farms

Reply:
yea no iron farms, i want to dissuade vanilla op strats at least, sorry for those who want their starting 4 iron